travis/build-wheels.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e -x
+
+yum install -y atlas-devel
+
+# Compile wheels
+for PYBIN in /opt/python/cp37-cp37m/bin; do
```

Comment:
Here and below on line 19 the code iterates over a list with one element, which serves no purpose. I presume that you're anticipating a time in which there will be several Python binary prefix paths and that you'll need to iterate over them; if that's the case, you should have a separate variable like `PYBIN_PATHS=/opt/python/cp37-cp37m/bin` and iterate over it on lines 7 and 19 with `for PYBIN in $PYBIN_PATHS;`. (You could set `PYBIN_PATHS` with `?=` if you want to optionally set it in the calling environment. You can use `tr ':' ' '` if you want `PYBIN_PATHS` to look like a normal colon-separated list of paths.)

Reply:
Additionally, I guess the `PYBIN` path come from the manylinux Docker image. It looks suspiciously specific. Is there really no more general path? Or is it encoding a Python version perhaps? If the latter, perhaps it's worth mentioning that?

Reply:
All good points. Will remove the looping and add a link to the manylinux project which talks about the paths